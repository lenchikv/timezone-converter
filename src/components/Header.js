import React from 'react';
import AppBar from '@material-ui/core/AppBar';

import Toolbar from '@material-ui/core/Toolbar';
import Typography from '@material-ui/core/Typography';
import IconButton from '@material-ui/core/IconButton';
import CompareArrows from '@material-ui/icons/CompareArrows';

const Header = () => (
  <div>
    <div className="appbarwrapper">
      <AppBar position="static">
        <Toolbar>
          <IconButton className="iconbuttonsyle" color="inherit" aria-label="Menu">
            <CompareArrows />
          </IconButton>
          <Typography variant="title" color="inherit" className="headertypoclass">
            React Timezone Converter
          </Typography>
        </Toolbar>
      </AppBar>
    </div>
  </div>
);

export default Header;
