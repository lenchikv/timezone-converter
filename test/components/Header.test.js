import { shallow } from 'enzyme';
import React from 'react';
import Header from '../../src/components/Header';

describe('Header', () => {
  it('renders correctly', () => {
    const tree = shallow(<Header />);
    expect(tree).toMatchSnapshot();
  });
});
