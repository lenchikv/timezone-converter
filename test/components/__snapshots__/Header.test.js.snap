// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div>
  <div
    className="appbarwrapper"
  >
    <WithStyles(AppBar)
      position="static"
    >
      <WithStyles(Toolbar)>
        <WithStyles(IconButton)
          aria-label="Menu"
          className="iconbuttonsyle"
          color="inherit"
        >
          <pure(CompareArrows) />
        </WithStyles(IconButton)>
        <WithStyles(Typography)
          className="headertypoclass"
          color="inherit"
          variant="title"
        >
          React Timezone Converter
        </WithStyles(Typography)>
      </WithStyles(Toolbar)>
    </WithStyles(AppBar)>
  </div>
</div>
`;
